src/background.js

```diff
@@ -66,6 +67,7 @@ const {
 	CDN_BASE_URL, BROWSER_INFO, IS_CLIQZ
 } = globals;
 const IS_EDGE = (BROWSER_INFO.name === 'edge');
+const IS_GHOSTERY_BROWSER = (BROWSER_INFO.name === 'ghostery_desktop');
```

Comment:
This is asynchronous. Check out `globals.BROWSER_INFO_READY` instead.

Reply:
Ah thanks, updated to await BROWSER_INFO_READY, however I see the browser info is being used before verifying that it is populated in a few other places which we should address in another PR -- we should also probably have BROWSER_INFO_READY short circuit if once a successful call to buildBrowserInfo is completed as currently every time we await `globals.BROWSER_INFO_READY` we are unnecessarily recalculating these values.

Reply:
Let's remove this line completely since it's evaluated at runtime and unreliable. The other browser_info checks should be working as is. The async check only applies to the desktop browser and Android browser to allow us to differentiate from stock FF.